mm/kfence/core.c

```diff
@@ -285,35 +254,34 @@ static bool __meminit kfence_allocate_pool(void)
 			pages[i].frozen = 1;
 		}
 	}
+
 	/* Skip the first page: it is reserved. */
+	// TODO(elver): Why is it reserved?
```

Comment:
No particular reason; initially the idea was to store metadata there. Right now it's because the pool always has an even number of pages (which we could also change, actually).

Reply:
Yeah, thanks. I think we clarified this earlier in the meeting. I'll add a comment explaining this and remove the TODOs.